src/overlay/SurveyManager.cpp

```diff
@@ -111,6 +112,10 @@ SurveyManager::startSurveyReporting()
     mCurve25519SecretKey = curve25519RandomSecret();
     mCurve25519PublicKey = curve25519DerivePublic(mCurve25519SecretKey);
 
+    mSurveyThrottleTimeoutMs =
```

Comment:
No need for this change (so we can keep SURVEY_THROTTLE_TIMEOUT_SEC a const). This throttling const is a very loose heuristic, so I don't think we need the extra complexity.

Reply:
Hmm it's actually been pretty challenging to get tests to work with the 5 second value. The tests all run simulation at 1 second close time, and I think the changes to tests are more complex than the actual survey change. Worst case we can address this as a follow up issue but I think it's kinda non-trivial

Reply:
Ok. This override is in the wrong place though (throttling begins before the reporting phase). So it's pretty confusing that the value changes mid-survey. I think the root cause is that `TARGET_LEDGER_CLOSE_TIME_BEFORE_PROTOCOL_VERSION_23_MS` doesn't handle accelerated time mode. You can solve that by adding a helper method to Config that returns either 5 or 1 second. Then the override can be removed.

Reply:
I've added a function that looks if test config flags are set. I decided on a local function just for survey manager instead of putting something in config. I don't want there to be 2 places globally to get the ledger close time, as this could be a potential footgun later on.